package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "6.5.12",
+  "version": "6.5.14",
```

Comment:
Wait, 2 jumps here? I think it should be `6.5.13`.

Reply:
Hongjing has a PR updating this to 13.

There's no easy solution here. If Hongjing merges before you then you'll need to rebase and you'll have a merge conflict. If you merge first then we skip a version (not a big deal).

**The main reason for the app version is so we can easily check in Dev/Test/Prod which version is running, so you can tell if your code is there or not.**

Reply:
Oh OK. I thought this PR is going to be merged first. Sorry about that, my bad! Resolving 👍

Reply:
This PR was ready before Hongjing's so I went ahead and merged it.

Leo, after the GitHub CD action has completed, verify this in Dev and then move your ticket to UX Assurance/Verification. I don't know if it's going to be possible to see this in action in Dev, but maybe the UX folks will be satisfied with a screenshot from you?

Reply:
PS ticket workflow:
- In Progress: you're working on it
- Review: your PR is ready and bring reviewed
- Staging: your PR has been merged and you have yet to verify in Dev
- UX Assurance/Verification: the UX folks want to look at it (denoted by UX Assurance label on ticket)
- Ready for QA: the UX folks didn't want to look at it, or they looked at it and are happy